assembly/pom.xml

```diff
@@ -24,6 +24,7 @@
 
   <modules>
     <module>client</module>
+    <module>integration</module>
```

Comment:
Should these be broken up further? I don't see a case for having Yarn and Mesos in the same assembly jar. Also, currently Yarn is not built by default, will this change work without -PYarn?